ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -95,7 +95,7 @@
             pattern="/etc/acs-commons/reports(.*)?"
             mode="merge"/>
         <include pattern="/var/acs-commons/httpcache"/>
```

Comment:
/var/acs-commons/mcp is missing as well, right?

Reply:
This will be added by #2045